src/pages/CardList/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useNavigate } from 'react-router-dom';
+
+import type { CardType } from '../../type';
+import Card from '../../components/Card';
+import Header from '../../components/Header';
+import './index.css';
+import { fetchLocalStorage } from '../../utils/applicationUtil';
+
+const CardListPage = () => {
+  const cardList = fetchLocalStorage('cardList', '[]');
+  const navigate = useNavigate();
+
+  const onAddButton = () => {
+    navigate('/add');
+  };
+
+  return (
+    <div className="card-list-page">
+      <Header>
+        <h3 className="card-list-page-header-title">보유카드</h3>
+      </Header>
+      <div className="card-list-page-body">
+        {cardList.length === 0 ? (
+          <span className="empty-card-list-title">새로운 카드를 등록해주세요.</span>
+        ) : (
+          cardList.map((card: CardType) => (
+            <div key={card.id}>
+              <Card
+                cardCompany={card.cardCompany}
+                cardFirstNumber={card.cardNumber.first}
+                cardSecondNumber={card.cardNumber.second}
+                cardThirdNumber={card.cardNumber.third}
+                cardFourthNumber={card.cardNumber.fourth}
+                cardOwner={card.cardOwner}
+                expireMonth={card.expireMonth}
+                expireYear={card.expireYear}
+              />
+              <p className="card-alias">{card.alias || card.cardCompany}</p>
+            </div>
+          ))
```

Comment:
```suggestion
          cardList.map(({ id, alias, ...rest }: CardType) => (
            <div key={id}>
              <Card {...rest} />
              <p className="card-alias">{alias || rest.cardCompany}</p>
            </div>
          ))
```

코드 줄이는 연습을 많이 해보는 걸 꼭 권장합니다😭